internal/components/components.go

```diff
@@ -155,7 +154,6 @@ func Get() (otelcol.Factories, error) {
 		oracledbreceiver.NewFactory(),
 		otlpreceiver.NewFactory(),
 		postgresqlreceiver.NewFactory(),
-		prometheusexecreceiver.NewFactory(),
```

Comment:
I believe you'll need to update the components_test.go and remove this receiver from the default component test.